Automattic/wp-calypso
# Update Preference Helper with Editing

#### Changes proposed in this Pull Request

* Update `NumberPreference`, `StringPreference`, and `BooleanPreference` to allow editing
* Update `ObjectPreference` to be composed of the above preferences
* Activate Preference helper on Green Calypso Dev

<img width="1210" alt="Screen Shot 2021-03-26 at 9 04 26 AM" src="https://user-images.githubusercontent.com/2810519/112660078-4863fd00-8e12-11eb-9449-bb26104cf652.png">

#### Testing instructions

1. Modify a preference and save
2. Verify the value has been saved
3. Modify a preference and reset
4. Verify the value has been reset to it's original
